admin/sql/updates/20240221-mbs-13492.sql

```diff
@@ -0,0 +1,22 @@
+\set ON_ERROR_STOP 1
+
+BEGIN;
+
+UPDATE editor
```

Comment:
I had a question about this query on IRC a while ago but I don't think it was addressed. Specifically the number of editors these conditions match seems to be unusually high:

```
musicbrainz_db=> SELECT count(*) FROM editor
 WHERE id != 4
   AND NOT deleted
   AND (
        member_since < NOW() - INTERVAL '2 weeks'
       OR
        NOT EXISTS (
          SELECT 1
            FROM edit
           WHERE edit.editor = editor.id
             AND edit.autoedit = 0
             AND edit.status = 2
          OFFSET 9
        ));
  count
---------
 2396138
(1 row)

musicbrainz_db=> select count(*) from editor;
  count
---------
 2402247
(1 row)
```
That would leave only 6,109 non-beginner editors. That doesn't seem right.